### ISSUE TITLE
C Compiler Hardwired to be clang

### ISSUE THREAD

**Issue by stevedonovan** [jonathandturner/peach#1]
So, the hack that I am, replaced 'clang' with 'gcc' in compiler.rs, line 680.
Things appeared to still work :)

**Comment by sophiajt** [comment]
Yeah, I *think* calling `cc` should be safe in unices. I was just looking at that's what the cc crate does (I wish it compiled binaries and not just libs, because I'd just use that...)
